src/components/Button/Button.tsx

```diff
@@ -19,44 +15,14 @@ export interface Props {
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
-  /**
-   * Toggles button that fills the full width of its container
-   */
-  fullWidth?: boolean;
-  /**
-   * Visually hide button text (but text is still accessible to assistive technology)
-   */
-  hideText?: boolean;
   /**
    * Link to URL. If href is present, the button will be rendered as an <a> element.
    */
   href?: string;
-  /**
-   * Name of SVG icon (i.e. caret-down, minus, warning)
-   */
-  iconName?: string;
-  /**
-   * Determines position of icon relative to button text.
-   * - **before** places icon before button text
-   * - **after** places icon after button text
-   */
-  iconPosition?: 'before' | 'after';
-  /**
-   * Button rendered on a dark backgorund
-   */
-  inverted?: boolean;
```

Comment:
We currently don't have any designs with light-colored buttons or links, and I don't think we even have any dark backgrounds in the app really (it's all white or light gray) *except* for the main sidebar navigation, which is a pretty unique case. We do have a similar prop in Along though because they do use dark backgrounds quite often.

Reply:
@dierat This is especially helpful for `link` and `bare` variants of a button, and can be useful for default and `primary` buttons that need to adjust their colors based on what backgrounds they sit against. I'd recommend keeping that in there, especially as the specifics of the design language are subject to change.

Reply:
I'll revert this change and ask design whether we want to support this feature in this component: https://app.shortcut.com/czi-edu/story/188471/button-api-questions-for-design